Makefile

```diff
@@ -41,10 +37,16 @@ ifeq (, $(shell which ineffassign))
 endif
 	ineffassign .
 
+lint:
+ifeq (, $(shell which golint))
+	$(error "golint not installed; you can install it with `go get -u golang.org/x/lint/golint`")
```

Comment:
note for future: we can vendor golint through go mod and run it from inside the vendor dir so people don't have to have a separate binary (or incompatible version)